src/main/Config.java

```diff
@@ -6,14 +6,14 @@
 
 public class Config {
 
-    public static final String LANGUAGE = PropertyFileReader.getInstance().getProperty(EConfig.LANGUAGE);
-    public static final String LASTPUSHEDDATE = PropertyFileReader.getInstance().getProperty(EConfig.LASTPUSHEDDATE);
-    public static final String STARSDECREASEAMOUNT = PropertyFileReader.getInstance().getProperty(EConfig.STARSDECREASEAMOUNT);
-    public static final BuildSystem BUILDSYSTEM = BuildSystem.getBuildType(PropertyFileReader.getInstance().getProperty(EConfig.BUILDSYSTEM));
-    public static final String OAUTHTOKEN = PropertyFileReader.getInstance().getProperty(EConfig.OAUTHTOKEN);
-    public static final String FILEPATH = PropertyFileReader.getInstance().getProperty(EConfig.FILEPATH);
-    public static final String JSONFILENAME = "repositories.json";
-    public static final String CUSTOMFILE = PropertyFileReader.getInstance().getProperty(EConfig.CUSTOMFILE);
+    public static final String LANGUAGE = null;//PropertyFileReader.getInstance().getProperty(EConfig.LANGUAGE);
+    public static final String LASTPUSHEDDATE = null;//PropertyFileReader.getInstance().getProperty(EConfig.LASTPUSHEDDATE);
+    public static final String STARSDECREASEAMOUNT = null;//PropertyFileReader.getInstance().getProperty(EConfig.STARSDECREASEAMOUNT);
+    public static final BuildSystem BUILDSYSTEM = null;//BuildSystem.getBuildType(PropertyFileReader.getInstance().getProperty(EConfig.BUILDSYSTEM));
+    public static final String OAUTHTOKEN = null;//PropertyFileReader.getInstance().getProperty(EConfig.OAUTHTOKEN);
+    public static final String FILEPATH = null;//PropertyFileReader.getInstance().getProperty(EConfig.FILEPATH);
+    public static final String JSONFILENAME = null;//"repositories.json";
+    public static final String CUSTOMFILE = null;//PropertyFileReader.getInstance().getProperty(EConfig.CUSTOMFILE);
```

Comment:
These properties are never set by the configservice.
 
Maybe add a getConfig() function to the ConfigurationService and call it here. That way we don't have to repeatedly parse the YAML file (as done currently by the getConfig() function in the configservice). 
Load it once at start up and set all properties in the CrawlerConfig.
This was the same approach as before with the PropertyFileReader.